interchange/LogicalNetlist.capnp

```diff
@@ -78,10 +85,17 @@ struct Netlist {
   }
 
   struct CellInstance {
-    name     @0 : StringIdx $stringRef();
-    propMap  @1 : PropertyMap;
-    view     @2 : StringIdx $stringRef();
-    cell     @3 : CellIdx $cellRef();
+    name       @0 : StringIdx $stringRef();
+    propMap    @1 : PropertyMap;
+    view       @2 : StringIdx $stringRef();
+    cell       @3 : CellIdx $cellRef();
+    placements @4 : List(CellInstancePlacement);
+  }
+  
+  # Acceptable physical placement for a cell instance
+  struct CellInstancePlacement {
+    siteType @0 : SiteTypeIdx $siteTypeRef;
+    bel      @1 : StringIdx $stringRef;
```

Comment:
There is a need to provide this information, but the `LogicalNetlist` would probably not be the place to do it.  The `LogicalNetlist` is meant to be device agnostic.  This information is device specific and would be better served to be placed in `DeviceResources`.

Reply:
If `LogicalNetlist` was meant to be device agnostic, but there's a need to store device-specific data related to entries in primitive libraries, then is it even a good container for primitive libraries?

A possible way out of this, without severely breaking backwards compatibility, would be to introduce a list field in `DeviceResources` that would have an entry for each instance in `instList` of `primLibs` which is not a pretty solution but at least it would move that out of `LogicalNetlist`. I'm not sure if an elegant solution is possible without changing too much about `primLibs`, because given your suggestion, we would have to store some data related to primitive libraries outside of the container that's supposed to encapsulate it.